.github/workflows/bootstrap.yaml

```diff
@@ -72,6 +72,8 @@ jobs:
     runs-on: ubuntu-latest
     # run only after lint finishes
     needs: lint
+    env:
+      REDIS_DISABLED: "azure s3"
```

Comment:
Don't disable caching for both, we still want some coverage of a non-local storage backend in combination with the cache.